examples/postprocess_dag.json

```diff
@@ -55,15 +58,33 @@
       "additional_attributes": {
         "nt_db": "s3://idseq-database/20170824/blast_db/nt"
       }
+    },
+    {
+      "in": ["fastqs", "amr_db"],
+      "out": "srst2_out",
+      "class": "PipelineStepRunSRST2",
+      "module": "idseq_dag.steps.run_srst2",
+      "additional_files": {"resist_gene_db": "s3://idseq-database/test/AMR/ARGannot_r2.fasta"},
+      "additional_attributes": {
+        "min_cov": 0,
+        "n_threads": 16,
+        "output_prefix": "output",
+        "db_name": "ARGannot_r2"
+      }
     }
   ],
   "given_targets": {
     "taxid_fasta_in": {
       "s3_dir": "s3://idseq-samples-prod/samples/12/5815/results/2.4"
-
     },
     "gsnap_m8": {
       "s3_dir": "s3://idseq-samples-prod/samples/12/5815/results/2.4"
-    }
+    },
+    "fastqs": {
+      "s3_dir":  "s3://idseq-samples-prod/samples/71/4542/fastqs", "count_reads": 1, "max_fragments": 75000000},
+    "amr_db": {
```

Comment:
Moving the reference to additional_files will also involve removing it from here